adcircpy/mesh/mesh.py

```diff
@@ -23,12 +24,12 @@ class AdcircMesh(EuclideanMesh2D):
     """
 
     def __init__(
-            self,
-            nodes,
-            elements,
-            boundaries=None,
-            crs=None,
-            description=None,
+        self,
```

Comment:
I thought we had decided to do double indentations on call signatures based on a PEP rule ?

Reply:
yes you're right, fixed